recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas_%.bbappend

```diff
@@ -24,12 +24,14 @@ SRC_URI_append_salvator-xs-h3-xt = " \
     file://r8a7795-salvator-xs-dom0.dts;subdir=git/arch/${ARCH}/boot/dts/renesas \
     file://r8a7795-salvator-xs-domd.dts;subdir=git/arch/${ARCH}/boot/dts/renesas \
     file://r8a7795-salvator-xs-doma.dts;subdir=git/arch/${ARCH}/boot/dts/renesas \
+    file://r8a7795-salvator-xs-domu.dts;subdir=git/arch/${ARCH}/boot/dts/renesas \
```

Comment:
I think, we need to reduce amount of copy-paste somehow.

Reply:
this is a refactoring which needs to be done later